active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,195 @@
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. 
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 the nodes have no option to open a drop-down menu, typically used to show the node's siblings.
+V2 will add this. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Represents a control that shows a list of items, typically the navigation trail to the current location.
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes.
+When there is not enough width to display the whole control,
+the leading nodes are replaced with a node that the user can click on,
+opening the missing nodes in a flyout menu.
+
+![Breadcrumb with nodes: Home, Documents, Design, Northwind, Images. The app is resized so that the Breadcrumb crumbles - then, clicking the ellipsis reveals the crumbled Home and Documents nodes](images/Breadcrumb_crumbling_flyout.gif)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Home", "Documents", "Design", "Northwind", "Images"}
+```
+
+If the last item in a Breadcrumb is text and is too long to fit it will be clipped
+(if the item type is a string it will be trimmed, meaning shortened and replaced with an ellipsis).
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+## Breadcrumb.DropDownItemTemplate
+
+Gets or sets the template to display the drop-down of nodes
+
+_Spec note: The casing should probably be "Dropdown" rather than "DropDown",
+but this is following existing precedent in Xaml, such as
+[ComboBox.IsDropDownOpen](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ComboBox.IsDropDownOpen)_.
+
+The Breadcrumb ItemTemplate is used to override the default display of the
+individual items in the Breadcrumb control.
+
+The DropDownItemTemplate can be used to override the default template used for
+the dropdown that is necessary when the control can't be wide enough and removes
+items from the front.
+
+![Diagram of the two Breadcrumb template properties](images/item-templates.jpg)
+
+The following configures a Breadcrumb so that it's dropdown displays bold text.
+
+```xaml
+<Breadcrumb>
+    <Breadcrumb.DropDownItemTemplate>
+        <DataTemplate x:DataType='BreadcrumbDropDownItem`>
+            <TextBlock Text='{x:Bind Content} FontWeight="Bold" />
```

Comment:
This is part of the item template, so making this bold would make all drop down items bold. Adding FontWeight here is just confusing i think.

Reply:
Yes, just demonstrating why the property exists and how it's used. And yes, FontWeight maybe isn't the best demonstration :). Can you think of another example?

Reply:
Perhaps adding an icon ?